### ISSUE TITLE
AWS::EC2::VPC supports referencing Route Table

### ISSUE THREAD

**Issue by Morras** [aws-cloudformation/aws-cloudformation-coverage-roadmap#357]
## 2. Scope of request 
AWS::EC2::VPC creates a default/main route table, but it is not possible to change the route table or edit the routs in the same template.  
By adding RouteTable to the Get::Attr, customers would be able to get a reference to the main route table and add routes to it.

#### In Create,  
Get::Attr RouteTable on the VPC should give you the ID of the main route table.
#### in Update, 
same as Create

## 5. Helpful Links to speed up research and evaluation

The premium support article for changing the main routes points to a custom resource that returns the main Route Table for the VPC. That should be native for the VPC resource: https://aws.amazon.com/premiumsupport/knowledge-center/cloudformation-route-table-vpc/

**Comment by tokohone** [comment]
Yes please, this is definitely needed since now when creating routes with CloudFormation template there's always the default RouteTable too even though it is redundant.
